api/institutions/authentication.py

```diff
@@ -109,7 +110,8 @@ def authenticate(self, request):
                 mimetype='html',
                 user=user,
                 domain=DOMAIN,
-                osf_support_email=OSF_SUPPORT_EMAIL
+                osf_support_email=OSF_SUPPORT_EMAIL,
+                storage_flag_is_active=storage_i18n_flag_active(),
```

Comment:
I think this will fail because you're accessing a flask thread-local in the Django app. Maybe just use `waffle.flag_is_active` directly here.

Reply:
Ah, good catch.